new/ohai-new-plugins-policy.md

```diff
@@ -0,0 +1,60 @@
+---
+RFC: unassigned
+Title: Ohai new plugins policy
+Author: Phil Dibowitz <phil@ipom.com>
+Status: Draft
+Type: <Standards Track, Informational, Process>
+---
+
+# Ohai new plugins policy
+
+To allow Chef to move fast while still providing stability, this lines out a
+policy for clarity.
+
+## Motivation
+
+    As a Chef developer,
+    I want to be able to provide new functionality to the community
+    so that they can use it.
+
+    As a Chef user,
+    I want to be able to leverage new functionality as quickly as possible
+    with minimal work while not breaking existing features.
+    
+    As a Chef user,
+    I also want to be able to discover available plugins.
```

Comment:
I think this is an important part of this RFC, but we're not mentioned it below. If we continue to add more plugins we really need to build the functionality to list installed plugins with descriptions. That should be included in this RFC